### PR TITLE
Added docstring of math operators of Variable

### LINKED ISSUE #1382: No documents on operators supported by Variable
Some python operators are supported by `Variable` (implemented in `functions/math/basic_math.py`) and some are not, but it is not clear from the current chainer documents. It would be nice if some documents are provided on supported operators.

### PR DESCRIPTION
**Pull request by unnonouno**
To prevent to show a first argument, I replaced names of arguments with `self`.
fix #1382

### CODE REVIEW

**Comment by gwtnb** [comment]
I think that it should be avoided to name an argument of non-method functions `self`. Additionally, it is helpful for users if the math operations description is on the reference page of the `Variable` class.

File: chainer/functions/math/basic_math.py [hunk 1]
```diff
@@ -276,11 +308,17 @@ def backward_gpu(self, x, gy):
             ''', 'div_bwd')(x[0], x[1], gy[0])
 
 
-def div(lhs, rhs):  # lhs / rhs
+def div(self, rhs):  # lhs / rhs
+    """Element-wise division
```
Comment (zori) [inline chainer/functions/math/basic_math.py]: it would be useful if this says "true division"

**Comment by unnonouno** [comment]
These methods are member methods for Variable, not for using directly (see `install_variable_arithmetics`). And documents of these method are genereted in Variable class.

**Comment by gwtnb** [comment]
> These methods are member methods for Variable, not for using directly (see install_variable_arithmetics).

I know.

> And documents of these method are genereted in Variable class.

Really? How did you build? When I built with sphinx and some other packages by `make html`, the methods were not generated.

**Comment by unnonouno** [comment]
I only executed 'make html', and it generated a document with the methods.

**Comment by gwtnb** [comment]
LGTM for Read the Docs.

**Pull request merged**